opennms-services/src/main/java/org/opennms/netmgt/poller/monitors/JMXMonitor.java

```diff
@@ -71,17 +72,15 @@
 
     private static final Logger LOG = LoggerFactory.getLogger(JMXMonitor.class);
 
-    private static final JexlEngine JEXL_ENGINE;
+    private static final OnmsJexlEngine JEXL_ENGINE;
     
     public static final String PARAM_BEAN_PREFIX = "beans.";
     public static final String PARAM_TEST_PREFIX = "tests.";
     public static final String PARAM_TEST = "test";
     public static final String PARAM_PORT = "port";
 
     static {
-        JEXL_ENGINE = new JexlEngine();
-        JEXL_ENGINE.setLenient(false);
-        JEXL_ENGINE.setStrict(true);
+        JEXL_ENGINE = new OnmsJexlEngine();
```

Comment:
I think we may need to whitelist `org.opennms.netmgt.poller.jmx.wrappers.ObjectNameWrapper` here.

Reply:
Done